superset/views/base.py

```diff
@@ -334,6 +334,30 @@ def render_app_template(self) -> FlaskResponse:
         )
 
 
+def get_environment_tag() -> dict[str, Any]:
+    # Whether flask is in debug mode (--debug)
+    debug = appbuilder.app.config.get("DEBUG")
+
+    # Getting the configuration option for ENVIRONMENT_TAG_CONFIG
+    env_tag_config = appbuilder.app.config.get("ENVIRONMENT_TAG_CONFIG")
```

Comment:
we assume config keys are always defined
`superset_app.config["ENVIRONMENT_TAG_CONFIG"]`